db/migrate/20200203190734_add_funder_and_org_to_plans.rb

```diff
@@ -0,0 +1,6 @@
+class AddFunderAndOrgToPlans < ActiveRecord::Migration
+  def change
+    add_reference :plans, :org, foreign_key: true
+    add_column :plans, :funder_id, :integer, index: true
```

Comment:
Is it worthwhile enforcing this as a foreign key to the orgs table, or is this sufficiently captured with our validation on the model?

Reply:
it seems to be working the same at least on the Rails side. Not sure if an fkey versus a regular index are different on the DB side.